packages/odyssey-storybook/src/components/Field/Field.mdx

```diff
@@ -1,43 +1,150 @@
+
+### Single Input
+
+Fields for lone inputs are rendered as `div's and assign their `label`/`input` relationship using an ID.
+
+<Canvas>
+  <Story id="components-textarea--default" />
+</Canvas>
+
+### Grouped Inputs
+
+Fields for grouped inputs are rendered as `fieldset`s, using a `legend` to label all of the associated `input`s accessibly. This variant also spreads hint and error associations across affected controls.
+
+<Canvas>
+  <Story id="components-radio--default" />
+</Canvas>
+
+## States
+
+Fields may be either required or optional, depending on what's called for by their input.
+
+### Required
+
+By default, inputs and Fields are set to `required`. As the default, this state has no unique affordance.
+
+<Canvas>
+  <Story id="components-textinput--text" />
+</Canvas>
+
+### Optional
+
+Fields with `required` set to `false` will be marked as optional:
+
+<Canvas>
+  <Story id="components-textinput--optional" />
+</Canvas>
 
 ## Usage
 
-Field labels provide textual affordances and context for Odyssey's various inputs. Our UI Kit provides these labels alongside each component.
+Field is a complementary component, providing shared structure across Odyssey Inputs like [Text Input](?path=/docs/components-textinput--text#text-input).
 
-Please refer to individual components for complete documentation. These examples are provided for quick illustration.
+When assembling Forms, it's unnecessary to utilize Field, as it's built into Odyssey's Inputs already.
 
-### Label
+If you are building a new or bespoke Input component, please utilize Field to ensure consistent behavior with existing components.
+
+## Content Guidelines
+
+### Labels
+
+A well-written label describes the purpose of the Field for the user.
+
+Use concise, descriptive labels (a word or two) to help users quickly scan your form.
+
+Finally, please use sentence casing for your labels. Sentence case is easier to read and will help reduce error rates and speed up form completion time.
+
+### Hints
+
+When writing hints, only provide information that assists with the immediate task.
+
+For example, "The expected format is `YYYY-MM-DD`."
+
+Hints like, "This information will be used to do x, y, and z." may be informative without assisting the user in completing their form.
+
+If additional disclosures or warnings are required, an [Infobox](?path=/docs/components-infobox--info#infobox) may be more appropriate.
 
-Labels caption pieces of UI, typically form fields. They are required for all inputs. When related to sets of Radios or Checkboxes, they may also be referred to as "legends".
+Remember: a good label may eliminate the need for a hint.
 
-Keep labels to a word or two so users can quickly scan the form. Always use sentence casing.
+### Errors
 
-**STORY MISSING**
+Error messaging should explain: (1) what has gone wrong and (2) how the user can fix the issue.
 
-### Field hint
+"This value is invalid." is less helpful than "This field requires a complete email address."
```

Comment:
the more helpful example here contradicts the guidelines above, it doesn't explain what has gone wrong. Should we reword this to instead say something like this?

Error messaging should at a minimum explain what has gone wrong and at best, explain how the user can fix the issue.